public/index.html

```diff
@@ -14,7 +14,7 @@ <h1 style="color: #559EFA; font-size: 60px; margin: 0">Triton VM</h1>
             Recursively verifiable<br/>
             STARKs for everyone.
         </p>
-        <a href="https://github.com/TritonVM/triton-vm" style="background-color: #559EFA; color: #191919; align-self: center; text-align: center; justify-items: center; font-size: 20px; padding: 10px; border-radius: 5px;">
+        <a href="https://github.com/TritonVM/triton-vm" style="background-color: #559EFA; color: #191919; align-self: center; text-align: center; justify-items: center; font-size: 20px; padding: 10px; border-radius: 5px; text-decoration: none;" href="https://github.com/TritonVM/triton-vm">
```

Comment:
The `href` tag is now duplicated, once at the very beginning and once at the very end of the line.